lib/vagrant-cloudstack/config.rb

```diff
@@ -417,6 +422,9 @@ def finalize!
         # private ip is nil by default
         @private_ip_address     = nil if @private_ip_address == UNSET_VALUE
 
+        # expunge on destroy is nil by default
+        @expunge_on_destroy     = nil if @expunge_on_destroy == UNSET_VALUE
```

Comment:
It would be nice if we could chek if this parameter does anything. But fog passes anything we add to options directly to cloudstack api